.github/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 <h3>New features since last release</h3>
 
+* [Placeholder for measurement grouping features]
```

Comment:
@ral9000 Can you propose a description for your new features in the changelog here?